CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.5.1 (2020-04-14)
+
+### Added
+Pin gevent to version 1.4.0
```

Comment:
Any clue why 1.5 is not working? Is it something on our side?
How do we know when we can unpin it?

Reply:
we are not sure. I am investigating if it could be an issue with talisker. But ATM so we can get a quick fix I pinned this and will start deploying it across sites.